platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/addressbook/RandomRosterBuilder.java

```diff
@@ -145,7 +148,9 @@ public Roster build() {
                     generateKeys(nodeId, addressBuilder);
                     return addressBuilder.build();
                 })
-                .toList());
+                .toList();
+
+        builder.rosterEntries(new ArrayList<>(entries)); // create a new list so that it is mutable
```

Comment:
Same as above. We shouldn't allow the mutability by the specification of this object.